packages/core/src/validate/run.ts

```diff
@@ -386,19 +386,19 @@ function* getStateVariableErrors(
 ): Generator<ValidationErrorWithName> {
   for (const varDecl of contractDef.nodes) {
     if (isNodeType('VariableDeclaration', varDecl)) {
```

Comment:
This looks odd but I just switch the two errors and chained them with `else if`.